add_sourceapp.sh

```diff
@@ -183,6 +185,12 @@ addlib() {
   esac
 }
 
+checkpermissions() {
+  apk="$1"
+  echo "Checking if any extra privapp-permissions are needed..."
+  python scripts/verify-permissions.py "$apk"
```

Comment:
This should be `python3`

Reply:
Line 31 with checktools also should check if `python3` is available

Reply:
Actually, I do have a comment! Can we return the exit code from the python script here (and not install the apk above if unsuccessful)?

Reply:
done and done